qiita_pet/support_files/doc/source/redbiom/redbiom.rst

```diff
@@ -0,0 +1,685 @@
+    well as *cry*.
+
+    N.B., the specific category in which a stem is found is not assured to be the same, *antibiot* could be in one category
+    and *cry* in another. A set intersection can be performed with "&", a union with "|" and a difference with "-".
+  * **Value search:** In addition to the stem-based search, value based searches can also be applied. These use a Python-like
+    grammar and allow for a rich set of comparisons to be performed based on a metadata category of interest. For example
+
+    .. code-block:: bash
+
+       where qiita_study_id == 10317
+
+    will find all samples which have the *qiita_study_id* metadata category, and in which the value for that sample is *10317*.
+
+  * **Examples:**
+
+    * Find all samples in which both the word 'infant', as well as 'antibiotics' exist, and where the infants are under a year old:
+
+    .. code-block:: bash
+
+       infant & antibiotics where age_years <= 1
+
+    * Find all samples only belonging to the EMP in which the pH is under 7, for a variety of sample types:
+
+      * soil:
+
+      .. code-block:: bash
+
+         soil where ph < 7 and emp_release1 == 'True'
+
+      * ocean water:
+
+      .. code-block:: bash
+
+         water & ocean where ph > 7 and emp_release1 == 'True'
+
+      * non-ocean water:
+
+      .. code-block:: bash
+
+         water - ocean where ph > 7 and emp_release1 == 'True'
+
+    * Or instead of pH you could search for a different metadata category:
+
+    .. code-block:: bash
+
+       water & ocean where salinity > 20
+
+    * Some other interesting examples:
+
+    .. code-block:: bash
+
+       feces & canine
+
+       (beer | cider | wine | alcohol)
+
+       where sample_type == 'stool'
+
+       usa where sample_type == 'stool' and host_taxid == 9606
+
+**2. Feature**:
+
+  * The search will be on all the features, in specific: **OTU ids for closed reference** or **exact sequences for deblur**.
+  * **Examples:**
+
+    * Find all samples in which the Greengenes feature 4479944 is found:
+
+    .. code-block:: bash
+
+       4479944
+
+    * Find all samples in which the sequence exists:
+
+    .. code-block:: bash
+
+       TACGAAGGGTGCAAGCATTACTCGGAATTACTGGGCGTAAAGCGTGCGTAGGTGGTTCGTTAAGTCTGATGTGAAAGCCCTGGGCTCAACCTGGGAACTG
+
+**3. Taxon**:
+
+  * The search will be **only on closed reference data** and based on the taxonomies available. Only exact matches are returned. Note that currently **only the Greengenes taxonomy** is searchable, and that it requires nomenclature of a rank prefix, two underscores, and then the name.
+
+  * **Examples:**
+
+    * Find all samples in which the genera Escherichia is found:
+
+    .. code-block:: bash
+
+       g__Escherichia
+
+    * Find all samples in which the phylum Tenericutes is found:
+
+    .. code-block:: bash
+
+       p__Tenericutes
+
+
+----------------------------------------
+
+
```

Comment:
I would suggest adding this as it's own thing, perhaps together with the `Statistical Analysis to Justify Clinical Trial Sample Size Tutorial`. I think being able to point directly to this vs. having it split in 2+ sections will make more sense. Note that during write up we split just so we could have concrete, achievable goals.

Reply:
Thank you for the feedback. 
I have added all three sections into a single document as suggested.